Resources/doc/reference/installation.rst

```diff
@@ -15,24 +15,39 @@ Installation
     //...
         "sonata-project/classification-bundle": "dev-master",
     //...
+    
+    },
+    "require-dev": {       
+        // ...
+        "sonata-project/easy-extends-bundle": "dev-master"
```

Comment:
Not sure but happy to change if needed. In the end I used  `"sonata-project/easy-extends-bundle":` "^2.2"`